description/description.rdfs.ttl

```diff
@@ -296,6 +338,16 @@ schema:copyrightYear a rdf:Property ;
     rdfs:comment "Het jaar waarin het auteursrecht voor het CreativeWork voor het eerst gold/geldend gemaakt werd."@nl ;
     rdfs:comment "L'année durant laquelle le droit d'auteur revendiqué pour l'œuvre créative a été revendiqué pour la première fois."@fr .
 
+schema:creator a rdf:Property ;
```

Comment:
akkoord, het is verwarrend, maar deze staat er al in als `schema:author`: https://github.com/viaacode/datamodels/blob/main/description/description.rdfs.ttl#L238